DemoApp/ChatPresenter.swift

```diff
@@ -14,7 +14,8 @@ class DemoChatChannelListRouter: ChatChannelListRouter {
     }
 
     var channelPresentingStyle: ChannelPresentingStyle = .push
-
+    var didLogout: (() -> Void)?
```

Comment:
nit: Usually, closures have `on` prefix 👍 
```suggestion
    var onLogout: (() -> Void)?
```